subsys/bluetooth/controller/hci_driver.c

```diff
@@ -533,6 +534,13 @@ static int configure_supported_features(void)
 		}
 	}
 
+#ifdef RADIO_TXP_DEFAULT
```

Comment:
No need to call the API if the default output power is 0.
```suggestion
#if RADIO_TXP_DEFAULT != 0
```
I also assume that RADIO_TXP_DEFAULT will always be defined